### PR TITLE
More juju 2.0 updates

### PR DESCRIPTION
**Pull request by lazypower**

### CODE REVIEW

File: install-review-tools.sh [hunk 1]
```diff
@@ -24,10 +26,15 @@ sudo apt-get install -qy \
...
+                        juju-deployer \
+                        python-jujuclient \
+                        make
+
 sudo pip install tox --upgrade
 
 echo "export LAYER_PATH=${HOME}/layers" >> /home/ubuntu/.bashrc
 echo "export INTERFACE_PATH=${HOME}/interfaces" >> /home/ubuntu/.bashrc
+# The builder defaults to pwd, this resets that behavior to output in  $HOME
+echo "export JUJU_REPOSITORY=${HOME}" >> /home/ubuntu/.bashrc
```
Comment (mbruzek) [inline install-review-tools.sh]: Replace with JUJU_DATA for 2.0

`JUJU_DATA=/home/ubuntu/.local/share/juju`

**Comment by mbruzek** [comment]
I think you need to set JUJU_DATA in the .bashrc

File: install-review-tools.sh (hunk 1 above)
Comment (lazypower) [inline install-review-tools.sh]: https://github.com/juju-solutions/jujubox/blob/devel/setup.sh#L24

**Comment by lazypower** [comment]
Sorry about that, its not obvious this lives in another layer

**Comment by mbruzek** [comment]
If this other area writes the .bashrc you should not echo these environment variables.

**Comment by mbruzek** [comment]
+1

**Pull request merged**